examples/web/vite-chat-app/tsconfig.node.json

```diff
@@ -4,7 +4,10 @@
     "skipLibCheck": true,
     "module": "ESNext",
     "moduleResolution": "bundler",
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    // requires the `override` keyword to override a method/property from a parent class
```

Comment:
Github highlighted this in red, presumably because comments aren't allowed in JSON. I'm not sure why it only flagged this one, but this comment should probably be removed wherever it appears in JSON.

Reply:
I can remove it here, I know that sometimes json does allow comments, but if you look at the default tsconfig file that gets generated for you by `tsc --init`, it actually contains lots of comments like [this](https://github.com/momentohq/client-sdk-javascript/pull/1283/files#diff-1dbbd8a9cf2a108111f1ad4a08c47f8af4934f44491121d86262e9b90f91a2b5R95-R98). I suspect that github doesn't recognize the `tsconfig.node.json` filename, and thinks its just a regular json file. I think removing it here is the correct thing to do since clearly github doesn't like it. But I do think the comment is useful, and the rest of the `tsconfig.json` files should be ok with comments in them

Reply:
Ah, ok, good to know. Thanks.